2019-03-29 10:07:14.314 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: aeda7f5285f74750856dfbdee4cba559 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:14.348 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: aeda7f5285f74750856dfbdee4cba559 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:18.435 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._reclaim_queued_deletes run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:07:18.436 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CONF.reclaim_instance_interval <= 0, skipping... _reclaim_queued_deletes /usr/lib/python2.7/site-packages/nova/compute/manager.py:6556
2019-03-29 10:07:19.435 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_rebooting_instances run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:07:21.435 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._instance_usage_audit run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:07:23.432 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._check_instance_build_time run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:07:23.435 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_rescued_instances run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:07:24.311 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 2fb773c9e9ec417590a2afe5857d31dd exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:24.343 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 2fb773c9e9ec417590a2afe5857d31dd __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:24.434 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager.update_available_resource run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:07:24.434 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 4498facdd67940fa84c8075042bdabc2 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:24.453 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 4498facdd67940fa84c8075042bdabc2 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:24.457 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Auditing locally available compute resources for com57 (node: com57) update_available_resource /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:534
2019-03-29 10:07:24.556 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Hypervisor: free VCPUs: 2 _report_hypervisor_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:655
2019-03-29 10:07:24.556 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Hypervisor/Node resource view: name=com57 free_ram=7160MB free_disk=44GB free_vcpus=2 pci_devices=[{"dev_id": "pci_0000_00_00_0", "product_id": "7190", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7190", "address": "0000:00:00.0"}, {"dev_id": "pci_0000_00_01_0", "product_id": "7191", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7191", "address": "0000:00:01.0"}, {"dev_id": "pci_0000_00_07_0", "product_id": "7110", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7110", "address": "0000:00:07.0"}, {"dev_id": "pci_0000_00_07_1", "product_id": "7111", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7111", "address": "0000:00:07.1"}, {"dev_id": "pci_0000_00_07_3", "product_id": "7113", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7113", "address": "0000:00:07.3"}, {"dev_id": "pci_0000_00_07_7", "product_id": "0740", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0740", "address": "0000:00:07.7"}, {"dev_id": "pci_0000_00_0f_0", "product_id": "0405", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0405", "address": "0000:00:0f.0"}, {"dev_id": "pci_0000_00_11_0", "product_id": "0790", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0790", "address": "0000:00:11.0"}, {"dev_id": "pci_0000_00_15_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.0"}, {"dev_id": "pci_0000_03_00_0", "product_id": "07c0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07c0", "address": "0000:03:00.0"}, {"dev_id": "pci_0000_00_15_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.1"}, {"dev_id": "pci_0000_00_15_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.2"}, {"dev_id": "pci_0000_00_15_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.3"}, {"dev_id": "pci_0000_00_15_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.4"}, {"dev_id": "pci_0000_00_15_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.5"}, {"dev_id": "pci_0000_00_15_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.6"}, {"dev_id": "pci_0000_00_15_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.7"}, {"dev_id": "pci_0000_00_16_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.0"}, {"dev_id": "pci_0000_0b_00_0", "product_id": "07b0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07b0", "address": "0000:0b:00.0"}, {"dev_id": "pci_0000_00_16_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.1"}, {"dev_id": "pci_0000_00_16_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.2"}, {"dev_id": "pci_0000_00_16_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.3"}, {"dev_id": "pci_0000_00_16_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.4"}, {"dev_id": "pci_0000_00_16_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.5"}, {"dev_id": "pci_0000_00_16_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.6"}, {"dev_id": "pci_0000_00_16_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.7"}, {"dev_id": "pci_0000_00_17_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.0"}, {"dev_id": "pci_0000_13_00_0", "product_id": "07b0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07b0", "address": "0000:13:00.0"}, {"dev_id": "pci_0000_00_17_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.1"}, {"dev_id": "pci_0000_00_17_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.2"}, {"dev_id": "pci_0000_00_17_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.3"}, {"dev_id": "pci_0000_00_17_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.4"}, {"dev_id": "pci_0000_00_17_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.5"}, {"dev_id": "pci_0000_00_17_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.6"}, {"dev_id": "pci_0000_00_17_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.7"}, {"dev_id": "pci_0000_00_18_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.0"}, {"dev_id": "pci_0000_00_18_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.1"}, {"dev_id": "pci_0000_00_18_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.2"}, {"dev_id": "pci_0000_00_18_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.3"}, {"dev_id": "pci_0000_00_18_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.4"}, {"dev_id": "pci_0000_00_18_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.5"}, {"dev_id": "pci_0000_00_18_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.6"}, {"dev_id": "pci_0000_00_18_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.7"}] _report_hypervisor_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:672
2019-03-29 10:07:24.557 10556 DEBUG oslo_concurrency.lockutils [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Lock "compute_resources" acquired by "nova.compute.resource_tracker._update_available_resource" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 10:07:24.559 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 329495b9161c439daa5d1bd2791f479c exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:24.589 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 329495b9161c439daa5d1bd2791f479c __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:24.591 10556 DEBUG nova.scheduler.client.report [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Refreshing aggregate associations for resource provider db732609-7867-486c-8e55-f52f70acfa62 _ensure_resource_provider /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:402
2019-03-29 10:07:24.628 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 83b8cee3494f4e4c965f018f18c5995a exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:24.643 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 83b8cee3494f4e4c965f018f18c5995a __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:24.658 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: c044b4e1638e4a66b531d3294fbc26c5 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:24.688 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: c044b4e1638e4a66b531d3294fbc26c5 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:24.689 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Total usable vcpus: 2, total allocated vcpus: 0 _report_final_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:688
2019-03-29 10:07:24.690 10556 INFO nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Final resource view: name=com57 phys_ram=8191MB used_ram=512MB phys_disk=47GB used_disk=0GB total_vcpus=2 used_vcpus=0 pci_stats=[]
2019-03-29 10:07:24.693 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Compute_service record updated for com57:com57 _update_available_resource /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:626
2019-03-29 10:07:24.694 10556 DEBUG oslo_concurrency.lockutils [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Lock "compute_resources" released by "nova.compute.resource_tracker._update_available_resource" :: held 0.137s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 10:07:24.695 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._heal_instance_info_cache run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:07:24.696 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Starting heal instance info cache _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5884
2019-03-29 10:07:24.696 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Rebuilding the list of instances to heal _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5888
2019-03-29 10:07:24.697 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: bb08893d7efe48f68122d70e01683d9a exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:24.720 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: bb08893d7efe48f68122d70e01683d9a __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:24.721 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Didn't find any instances for network info cache update. _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5960
2019-03-29 10:07:25.460 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_unconfirmed_resizes run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:07:26.435 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_volume_usage run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:07:29.252 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received message msg_id: ae9695aa51c44a28897d3e268b344c13 reply to reply_d82da3782cfe4e3aac96d8380642efc7 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:255
2019-03-29 10:07:29.255 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 73fa928a110343a790a7a8013ca850e6 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:29.286 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 73fa928a110343a790a7a8013ca850e6 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:29.287 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 851215af99a84e3a9f45783a1d90bbd4 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:29.301 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 851215af99a84e3a9f45783a1d90bbd4 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:29.303 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 075f22c6bd3344f981e45cacfc403ec9 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:29.314 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 075f22c6bd3344f981e45cacfc403ec9 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:29.316 10556 DEBUG nova.objects.instance [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lazy-loading 'vcpu_model' on Instance uuid f5a19c94-1a45-4ea0-9386-77c10705c825 obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 10:07:29.317 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: c07c9a0e25d84158b5dae5dac35caf4c exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:29.425 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: c07c9a0e25d84158b5dae5dac35caf4c __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:29.426 10556 DEBUG nova.virt.libvirt.driver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Creating tmpfile /var/lib/nova/instances/tmpdjYwQ4 to notify to other compute nodes that they should mount the same storage. _create_shared_storage_test_file /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6033
2019-03-29 10:07:29.427 10556 DEBUG nova.compute.manager [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] destination check data is LibvirtLiveMigrateData(bdms=<?>,block_migration=True,disk_available_mb=45056,disk_over_commit=False,filename='tmpdjYwQ4',graphics_listen_addr_spice=127.0.0.1,graphics_listen_addr_vnc=0.0.0.0,image_type='default',instance_relative_path=<?>,is_shared_block_storage=<?>,is_shared_instance_path=<?>,is_volume_backed=<?>,migration=<?>,serial_listen_addr=None,serial_listen_ports=<?>,supported_perf_events=<?>,target_connect_addr=<?>) _do_check_can_live_migrate_destination /usr/lib/python2.7/site-packages/nova/compute/manager.py:5272
2019-03-29 10:07:29.429 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 837ef009490343c7b5bab9fc114c2993 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:29.440 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 837ef009490343c7b5bab9fc114c2993 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:29.444 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 55e7b3e8d927480d874d71c13c6776fe exchange 'nova' topic 'compute.com58' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:29.538 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 55e7b3e8d927480d874d71c13c6776fe __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:29.539 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 382d745389164fdcbff3c137f86cfff9 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:29.573 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 382d745389164fdcbff3c137f86cfff9 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:29.575 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] sending reply msg_id: ae9695aa51c44a28897d3e268b344c13 reply queue: reply_d82da3782cfe4e3aac96d8380642efc7 time elapsed: 0.322092966991s _send_reply /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:130
2019-03-29 10:07:29.869 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received message msg_id: 15077abfce144467997b439fb7a67be2 reply to reply_b16e68cc3da145ef95ee7086e953eac7 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:255
2019-03-29 10:07:29.874 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 67eb6756d93f4ae1837ced804b1220f2 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:29.911 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 67eb6756d93f4ae1837ced804b1220f2 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:29.912 10556 DEBUG nova.compute.manager [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] pre_live_migration data is LibvirtLiveMigrateData(bdms=<?>,block_migration=True,disk_available_mb=45056,disk_over_commit=False,filename='tmpdjYwQ4',graphics_listen_addr_spice=127.0.0.1,graphics_listen_addr_vnc=0.0.0.0,image_type='default',instance_relative_path='f5a19c94-1a45-4ea0-9386-77c10705c825',is_shared_block_storage=False,is_shared_instance_path=False,is_volume_backed=True,migration=<?>,serial_listen_addr=None,serial_listen_ports=<?>,supported_perf_events=<?>,target_connect_addr=<?>) pre_live_migration /usr/lib/python2.7/site-packages/nova/compute/manager.py:5332
2019-03-29 10:07:29.914 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: c94f3662d6f740dbba7dc285d8130eb8 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:29.944 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: c94f3662d6f740dbba7dc285d8130eb8 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:29.946 10556 DEBUG os_brick.utils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] ==> get_connector_properties: call u"{'execute': None, 'my_ip': '172.30.126.57', 'enforce_multipath': True, 'host': 'com57', 'root_helper': 'sudo nova-rootwrap /etc/nova/rootwrap.conf', 'multipath': False}" trace_logging_wrapper /usr/lib/python2.7/site-packages/os_brick/utils.py:146
2019-03-29 10:07:29.947 10556 INFO oslo.privsep.daemon [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Running privsep helper: ['sudo', 'nova-rootwrap', '/etc/nova/rootwrap.conf', 'privsep-helper', '--config-file', '/usr/share/nova/nova-dist.conf', '--config-file', '/etc/nova/nova.conf', '--privsep_context', 'os_brick.privileged.default', '--privsep_sock_path', '/tmp/tmpcJWHZs/privsep.sock']
2019-03-29 10:07:30.633 10556 INFO oslo.privsep.daemon [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Spawned new privsep daemon via rootwrap
2019-03-29 10:07:30.634 10556 DEBUG oslo.privsep.daemon [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Accepted privsep connection to /tmp/tmpcJWHZs/privsep.sock __init__ /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:313
2019-03-29 10:07:30.636 10556 INFO oslo.privsep.daemon [-] privsep daemon starting
2019-03-29 10:07:30.636 10556 INFO oslo.privsep.daemon [-] privsep process running with uid/gid: 0/0
2019-03-29 10:07:30.637 10556 INFO oslo.privsep.daemon [-] privsep process running with capabilities (eff/prm/inh): CAP_SYS_ADMIN/CAP_SYS_ADMIN/none
2019-03-29 10:07:30.637 10556 INFO oslo.privsep.daemon [-] privsep daemon running as pid 10664
2019-03-29 10:07:30.637 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (1,) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:30.638 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (2,) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:30.641 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'os_brick.privileged.rootwrap.execute_root', ('cat', '/etc/iscsi/initiatorname.iscsi'), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:30.709 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): cat /etc/iscsi/initiatorname.iscsi out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:30.713 10556 DEBUG oslo.privsep.daemon [-] CMD "cat /etc/iscsi/initiatorname.iscsi" returned: 0 in 0.004s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:30.714 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, ('InitiatorName=iqn.1994-05.com.redhat:4d12e79a68be\n', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:30.715 10556 DEBUG os_brick.initiator.linuxfc [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No Fibre Channel support detected on system. get_fc_hbas /usr/lib/python2.7/site-packages/os_brick/initiator/linuxfc.py:91
2019-03-29 10:07:30.715 10556 DEBUG os_brick.initiator.linuxfc [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No Fibre Channel support detected on system. get_fc_hbas /usr/lib/python2.7/site-packages/os_brick/initiator/linuxfc.py:91
2019-03-29 10:07:30.716 10556 DEBUG os_brick.utils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] <== get_connector_properties: return (769ms) {'initiator': u'iqn.1994-05.com.redhat:4d12e79a68be', 'ip': u'172.30.126.57', 'platform': u'x86_64', 'host': u'com57', 'do_local_attach': False, 'os_type': u'linux2', 'multipath': False} trace_logging_wrapper /usr/lib/python2.7/site-packages/os_brick/utils.py:170
2019-03-29 10:07:30.726 10556 DEBUG cinderclient.v2.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] REQ: curl -g -i -X POST http://172.30.126.56:8776/v2/d4bbd972ca1b463ba028ccd1df983917/volumes/78a56ddc-ee21-4278-991e-17b30d68fa6e/action -H "User-Agent: python-cinderclient" -H "Content-Type: application/json" -H "Accept: application/json" -H "X-Auth-Token: {SHA1}8909f64cb46a118d57e7cdb53d3762c4971c4edf" -d '{"os-initialize_connection": {"connector": {"platform": "x86_64", "host": "com57", "do_local_attach": false, "ip": "172.30.126.57", "os_type": "linux2", "multipath": false, "initiator": "iqn.1994-05.com.redhat:4d12e79a68be"}}}' _http_log_request /usr/lib/python2.7/site-packages/keystoneauth1/session.py:347
2019-03-29 10:07:34.315 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: e3eb740d7784434689b93cb39e3565dd exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:34.355 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: e3eb740d7784434689b93cb39e3565dd __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:34.539 10556 DEBUG cinderclient.v2.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] RESP: [200] X-Compute-Request-Id: req-0d8d8159-5487-4333-9bbd-22cdb6f258df Content-Type: application/json Content-Length: 447 X-Openstack-Request-Id: req-0d8d8159-5487-4333-9bbd-22cdb6f258df Date: Fri, 29 Mar 2019 02:07:34 GMT Connection: keep-alive 
RESP BODY: {"connection_info": {"driver_volume_type": "iscsi", "data": {"auth_password": "TPnqk7LML2MKpPv4", "target_discovered": false, "encrypted": false, "qos_specs": null, "target_iqn": "iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e", "target_portal": "172.30.126.56:3260", "volume_id": "78a56ddc-ee21-4278-991e-17b30d68fa6e", "target_lun": 0, "access_mode": "rw", "auth_username": "PutS9jPtTsLadXTr42qr", "auth_method": "CHAP"}}}
 _http_log_response /usr/lib/python2.7/site-packages/keystoneauth1/session.py:395
2019-03-29 10:07:34.539 10556 DEBUG cinderclient.v2.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] POST call to cinderv2 for http://172.30.126.56:8776/v2/d4bbd972ca1b463ba028ccd1df983917/volumes/78a56ddc-ee21-4278-991e-17b30d68fa6e/action used request id req-0d8d8159-5487-4333-9bbd-22cdb6f258df request /usr/lib/python2.7/site-packages/keystoneauth1/session.py:640
2019-03-29 10:07:34.541 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 542b69daa6ac43f282027e8fa1c88d07 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:34.570 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 542b69daa6ac43f282027e8fa1c88d07 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:34.571 10556 DEBUG oslo_concurrency.lockutils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Acquired semaphore "refresh_cache-f5a19c94-1a45-4ea0-9386-77c10705c825" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2019-03-29 10:07:34.571 10556 DEBUG nova.network.neutronv2.api [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] _get_instance_nw_info() _get_instance_nw_info /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1295
2019-03-29 10:07:34.572 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 634206d820aa47558409e6a1b478a70c exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:34.580 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 634206d820aa47558409e6a1b478a70c __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:35.151 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=f5a19c94-1a45-4ea0-9386-77c10705c825 used request id req-ddcafc63-63d1-4501-a5dc-d440d817caa7 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:07:35.294 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/networks.json?id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d used request id req-07e05fe8-e56d-4a3f-8ef7-c88d6417ccf8 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:07:35.316 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/floatingips.json?fixed_ip_address=10.10.10.11&port_id=b3249515-3aab-439f-ba9c-bc498756871a used request id req-d6557ece-8432-482d-895b-7f25901ea0e7 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:07:35.420 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/subnets.json?id=b9f5b815-a0ab-4476-a216-6b824fda2957 used request id req-fe4bf833-b2ba-4362-af33-d1fdd17e9110 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:07:35.462 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?network_id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d&device_owner=network%3Adhcp used request id req-a83d82f1-2ca3-4bd8-87eb-a277cb6d1451 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:07:35.463 10556 DEBUG nova.network.base_api [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Updating instance_info_cache with network_info: [{"profile": {}, "ovs_interfaceid": "b3249515-3aab-439f-ba9c-bc498756871a", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.11"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tapb3249515-3a", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:29:72:d6", "active": true, "type": "ovs", "id": "b3249515-3aab-439f-ba9c-bc498756871a", "qbg_params": null}] update_instance_cache_with_nw_info /usr/lib/python2.7/site-packages/nova/network/base_api.py:48
2019-03-29 10:07:35.464 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 85151b442cd64c1ba2dfad9b995f5bb6 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:35.478 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 85151b442cd64c1ba2dfad9b995f5bb6 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:35.479 10556 DEBUG oslo_concurrency.lockutils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Releasing semaphore "refresh_cache-f5a19c94-1a45-4ea0-9386-77c10705c825" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2019-03-29 10:07:35.479 10556 WARNING nova.image.glance [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 10:07:35.479 10556 DEBUG nova.objects.instance [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lazy-loading 'flavor' on Instance uuid f5a19c94-1a45-4ea0-9386-77c10705c825 obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 10:07:35.481 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 73b2f75383bc4c96827af064c4f481ac exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:35.532 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 73b2f75383bc4c96827af064c4f481ac __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:35.534 10556 DEBUG nova.virt.libvirt.driver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] migrate_data in pre_live_migration: LibvirtLiveMigrateData(bdms=<?>,block_migration=True,disk_available_mb=45056,disk_over_commit=False,filename='tmpdjYwQ4',graphics_listen_addr_spice=127.0.0.1,graphics_listen_addr_vnc=0.0.0.0,image_type='default',instance_relative_path='f5a19c94-1a45-4ea0-9386-77c10705c825',is_shared_block_storage=False,is_shared_instance_path=False,is_volume_backed=True,migration=<?>,serial_listen_addr=None,serial_listen_ports=<?>,supported_perf_events=<?>,target_connect_addr=<?>) pre_live_migration /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6710
2019-03-29 10:07:35.535 10556 DEBUG nova.virt.libvirt.driver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Creating instance directory: /var/lib/nova/instances/f5a19c94-1a45-4ea0-9386-77c10705c825 pre_live_migration /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6723
2019-03-29 10:07:35.535 10556 DEBUG nova.virt.libvirt.driver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Creating disk.info with the contents: {u'/var/lib/nova/instances/f5a19c94-1a45-4ea0-9386-77c10705c825/disk.config': u'raw'} pre_live_migration /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6737
2019-03-29 10:07:35.536 10556 DEBUG nova.virt.libvirt.driver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Checking to make sure images and backing files are present before live migration. pre_live_migration /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6747
2019-03-29 10:07:35.536 10556 DEBUG oslo_concurrency.processutils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Running cmd (subprocess): qemu-img create -f raw /var/lib/nova/instances/f5a19c94-1a45-4ea0-9386-77c10705c825/disk.config 468992 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:355
2019-03-29 10:07:35.571 10556 DEBUG oslo_concurrency.processutils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CMD "qemu-img create -f raw /var/lib/nova/instances/f5a19c94-1a45-4ea0-9386-77c10705c825/disk.config 468992" returned: 0 in 0.034s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:385
2019-03-29 10:07:35.572 10556 DEBUG nova.virt.libvirt.volume.remotefs [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Copying file com58:/var/lib/nova/instances/f5a19c94-1a45-4ea0-9386-77c10705c825/disk.config to /var/lib/nova/instances/f5a19c94-1a45-4ea0-9386-77c10705c825 copy_file /usr/lib/python2.7/site-packages/nova/virt/libvirt/volume/remotefs.py:112
2019-03-29 10:07:35.573 10556 DEBUG oslo_concurrency.processutils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Running cmd (subprocess): scp -r com58:/var/lib/nova/instances/f5a19c94-1a45-4ea0-9386-77c10705c825/disk.config /var/lib/nova/instances/f5a19c94-1a45-4ea0-9386-77c10705c825 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:355
2019-03-29 10:07:35.940 10556 DEBUG oslo_concurrency.processutils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CMD "scp -r com58:/var/lib/nova/instances/f5a19c94-1a45-4ea0-9386-77c10705c825/disk.config /var/lib/nova/instances/f5a19c94-1a45-4ea0-9386-77c10705c825" returned: 0 in 0.367s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:385
2019-03-29 10:07:35.941 10556 DEBUG nova.virt.libvirt.driver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Connecting volumes before live migration. pre_live_migration /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6784
2019-03-29 10:07:35.942 10556 DEBUG nova.virt.libvirt.volume.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Calling os-brick to attach iSCSI Volume connect_volume /usr/lib/python2.7/site-packages/nova/virt/libvirt/volume/iscsi.py:63
2019-03-29 10:07:35.943 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] ==> connect_volume: call u"{'args': (<os_brick.initiator.connectors.iscsi.ISCSIConnector object at 0x596ea50>, {u'access_mode': u'rw', u'target_discovered': False, u'encrypted': False, u'qos_specs': None, u'target_iqn': u'iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e', u'target_portal': u'172.30.126.56:3260', u'volume_id': u'78a56ddc-ee21-4278-991e-17b30d68fa6e', u'target_lun': 0, u'auth_password': u'***', u'auth_username': u'PutS9jPtTsLadXTr42qr', u'auth_method': u'CHAP'}), 'kwargs': {}}" trace_logging_wrapper /usr/lib/python2.7/site-packages/os_brick/utils.py:146
2019-03-29 10:07:35.943 10556 DEBUG oslo_concurrency.lockutils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "connect_volume" acquired by "os_brick.initiator.connectors.iscsi.connect_volume" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 10:07:35.944 10556 INFO os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Multipath discovery for iSCSI not enabled.
2019-03-29 10:07:35.944 10556 INFO os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Trying to connect to iSCSI portal 172.30.126.56:3260
2019-03-29 10:07:35.945 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'os_brick.privileged.rootwrap.execute_root', ('iscsiadm', '-m', 'node', '-T', u'iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e', '-p', u'172.30.126.56:3260'), {'attempts': 1, 'check_exit_code': 0, 'delay_on_retry': True}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.945 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.952 10556 DEBUG oslo.privsep.daemon [-] CMD "iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260" returned: 21 in 0.007s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.952 10556 DEBUG oslo.privsep.daemon [-] u'iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260' failed. Not Retrying. out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.953 10556 DEBUG oslo.privsep.daemon [-] privsep: Exception during request[77878608]: Unexpected error while running command.
Command: iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260
Exit code: 21
Stdout: u''
Stderr: u'iscsiadm: No records found\n' out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.953 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (5, 'oslo_concurrency.processutils.ProcessExecutionError', (u'', u'iscsiadm: No records found\n', 21, u'iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260', None)) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.954 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'os_brick.privileged.rootwrap.execute_root', ('iscsiadm', '-m', 'node', '-T', u'iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e', '-p', u'172.30.126.56:3260', '--interface', 'default', '--op', 'new'), {'attempts': 1, 'check_exit_code': 0, 'delay_on_retry': True}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.954 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --interface default --op new out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.959 10556 DEBUG oslo.privsep.daemon [-] CMD "iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --interface default --op new" returned: 0 in 0.005s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.960 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, ('New iSCSI node [tcp:[hw=,ip=,net_if=,iscsi_if=default] 172.30.126.56,3260,-1 iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e] added\n', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.960 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] iscsiadm ('--interface', 'default', '--op', 'new'): stdout=New iSCSI node [tcp:[hw=,ip=,net_if=,iscsi_if=default] 172.30.126.56,3260,-1 iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e] added
 stderr= _run_iscsiadm /usr/lib/python2.7/site-packages/os_brick/initiator/connectors/iscsi.py:598
2019-03-29 10:07:35.961 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'os_brick.privileged.rootwrap.execute_root', ('iscsiadm', '-m', 'node', '-T', u'iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e', '-p', u'172.30.126.56:3260', '--op', 'update', '-n', 'node.session.auth.authmethod', '-v', u'CHAP'), {'attempts': 1, 'check_exit_code': 0, 'delay_on_retry': True}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.961 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --op update -n node.session.auth.authmethod -v CHAP out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.965 10556 DEBUG oslo.privsep.daemon [-] CMD "iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --op update -n node.session.auth.authmethod -v CHAP" returned: 0 in 0.004s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.966 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, ('', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.966 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] iscsiadm ('--op', 'update', '-n', 'node.session.auth.authmethod', '-v', u'CHAP'): stdout= stderr= _run_iscsiadm /usr/lib/python2.7/site-packages/os_brick/initiator/connectors/iscsi.py:598
2019-03-29 10:07:35.966 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'os_brick.privileged.rootwrap.execute_root', ('iscsiadm', '-m', 'node', '-T', u'iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e', '-p', u'172.30.126.56:3260', '--op', 'update', '-n', 'node.session.auth.username', '-v', u'PutS9jPtTsLadXTr42qr'), {'attempts': 1, 'check_exit_code': 0, 'delay_on_retry': True}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.967 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --op update -n node.session.auth.username -v PutS9jPtTsLadXTr42qr out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.974 10556 DEBUG oslo.privsep.daemon [-] CMD "iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --op update -n node.session.auth.username -v PutS9jPtTsLadXTr42qr" returned: 0 in 0.007s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.974 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, ('', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.974 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] iscsiadm ('--op', 'update', '-n', 'node.session.auth.username', '-v', u'PutS9jPtTsLadXTr42qr'): stdout= stderr= _run_iscsiadm /usr/lib/python2.7/site-packages/os_brick/initiator/connectors/iscsi.py:598
2019-03-29 10:07:35.975 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'os_brick.privileged.rootwrap.execute_root', ('iscsiadm', '-m', 'node', '-T', u'iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e', '-p', u'172.30.126.56:3260', '--op', 'update', '-n', 'node.session.auth.password', '-v', u'TPnqk7LML2MKpPv4'), {'attempts': 1, 'check_exit_code': 0, 'delay_on_retry': True}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.975 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --op update -n node.session.auth.password -v *** out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.991 10556 DEBUG oslo.privsep.daemon [-] CMD "iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --op update -n node.session.auth.password -v ***" returned: 0 in 0.015s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.991 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, ('', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.991 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] iscsiadm ('--op', 'update', '-n', 'node.session.auth.password', '-v', u'***'): stdout= stderr= _run_iscsiadm /usr/lib/python2.7/site-packages/os_brick/initiator/connectors/iscsi.py:598
2019-03-29 10:07:35.992 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'os_brick.privileged.rootwrap.execute_root', ('iscsiadm', '-m', 'session'), {'check_exit_code': (0, 1, 21)}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.992 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): iscsiadm -m session out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.996 10556 DEBUG oslo.privsep.daemon [-] CMD "iscsiadm -m session" returned: 21 in 0.004s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.996 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, ('', 'iscsiadm: No active sessions.\n')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.996 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] iscsiadm ['-m', 'session']: stdout= stderr=iscsiadm: No active sessions.
 _run_iscsiadm_bare /usr/lib/python2.7/site-packages/os_brick/initiator/connectors/iscsi.py:824
2019-03-29 10:07:35.997 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'os_brick.privileged.rootwrap.execute_root', ('iscsiadm', '-m', 'node', '-T', u'iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e', '-p', u'172.30.126.56:3260', '--login'), {'attempts': 1, 'check_exit_code': (0, 255), 'delay_on_retry': True}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:35.997 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --login out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.042 10556 DEBUG oslo.privsep.daemon [-] CMD "iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --login" returned: 0 in 0.044s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.042 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, ('Logging in to [iface: default, target: iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e, portal: 172.30.126.56,3260] (multiple)\nLogin to [iface: default, target: iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e, portal: 172.30.126.56,3260] successful.\n', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.043 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] iscsiadm ('--login',): stdout=Logging in to [iface: default, target: iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e, portal: 172.30.126.56,3260] (multiple)
Login to [iface: default, target: iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e, portal: 172.30.126.56,3260] successful.
 stderr= _run_iscsiadm /usr/lib/python2.7/site-packages/os_brick/initiator/connectors/iscsi.py:598
2019-03-29 10:07:36.045 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'os_brick.privileged.rootwrap.execute_root', ('iscsiadm', '-m', 'node', '-T', u'iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e', '-p', u'172.30.126.56:3260', '--op', 'update', '-n', 'node.startup', '-v', 'automatic'), {'attempts': 1, 'check_exit_code': 0, 'delay_on_retry': True}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.045 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --op update -n node.startup -v automatic out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.049 10556 DEBUG oslo.privsep.daemon [-] CMD "iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --op update -n node.startup -v automatic" returned: 0 in 0.005s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.050 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, ('', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.050 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] iscsiadm ('--op', 'update', '-n', 'node.startup', '-v', 'automatic'): stdout= stderr= _run_iscsiadm /usr/lib/python2.7/site-packages/os_brick/initiator/connectors/iscsi.py:598
2019-03-29 10:07:36.050 10556 INFO os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] ISCSI volume not yet found at: [u'/dev/disk/by-path/ip-172.30.126.56:3260-iscsi-iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e-lun-0']. Will rescan & retry.  Try number: 0.
2019-03-29 10:07:36.052 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'os_brick.privileged.rootwrap.execute_root', ('iscsiadm', '-m', 'node', '-T', u'iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e', '-p', u'172.30.126.56:3260', '--rescan'), {'attempts': 1, 'check_exit_code': 0, 'delay_on_retry': True}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.053 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --rescan out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.070 10556 DEBUG oslo.privsep.daemon [-] CMD "iscsiadm -m node -T iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e -p 172.30.126.56:3260 --rescan" returned: 0 in 0.017s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.070 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, ('Rescanning session [sid: 11, target: iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e, portal: 172.30.126.56,3260]\n', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.071 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] iscsiadm ('--rescan',): stdout=Rescanning session [sid: 11, target: iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e, portal: 172.30.126.56,3260]
 stderr= _run_iscsiadm /usr/lib/python2.7/site-packages/os_brick/initiator/connectors/iscsi.py:598
2019-03-29 10:07:36.071 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Found iSCSI node [u'/dev/disk/by-path/ip-172.30.126.56:3260-iscsi-iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e-lun-0'] (after 1 rescans) connect_volume /usr/lib/python2.7/site-packages/os_brick/initiator/connectors/iscsi.py:437
2019-03-29 10:07:36.073 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'os_brick.privileged.rootwrap.execute_root', ('/lib/udev/scsi_id', '--page', '0x83', '--whitelisted', u'/dev/disk/by-path/ip-172.30.126.56:3260-iscsi-iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e-lun-0'), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.073 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): /lib/udev/scsi_id --page 0x83 --whitelisted /dev/disk/by-path/ip-172.30.126.56:3260-iscsi-iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e-lun-0 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.078 10556 DEBUG oslo.privsep.daemon [-] CMD "/lib/udev/scsi_id --page 0x83 --whitelisted /dev/disk/by-path/ip-172.30.126.56:3260-iscsi-iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e-lun-0" returned: 0 in 0.005s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.078 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, ('36001405fb81b00707464d729f24ff6c3\n', '')) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.078 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Device WWN = '36001405fb81b00707464d729f24ff6c3' connect_volume /usr/lib/python2.7/site-packages/os_brick/initiator/connectors/iscsi.py:444
2019-03-29 10:07:36.079 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] connect_volume returning {'path': u'/dev/disk/by-path/ip-172.30.126.56:3260-iscsi-iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e-lun-0', 'scsi_wwn': u'36001405fb81b00707464d729f24ff6c3', 'type': 'block'} connect_volume /usr/lib/python2.7/site-packages/os_brick/initiator/connectors/iscsi.py:456
2019-03-29 10:07:36.079 10556 DEBUG oslo_concurrency.lockutils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Lock "connect_volume" released by "os_brick.initiator.connectors.iscsi.connect_volume" :: held 0.136s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 10:07:36.080 10556 DEBUG os_brick.initiator.connectors.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] <== connect_volume: return (136ms) {'path': u'/dev/disk/by-path/ip-172.30.126.56:3260-iscsi-iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e-lun-0', 'scsi_wwn': u'36001405fb81b00707464d729f24ff6c3', 'type': u'block'} trace_logging_wrapper /usr/lib/python2.7/site-packages/os_brick/utils.py:170
2019-03-29 10:07:36.080 10556 DEBUG nova.virt.libvirt.volume.iscsi [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Attached iSCSI volume {'path': u'/dev/disk/by-path/ip-172.30.126.56:3260-iscsi-iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e-lun-0', 'scsi_wwn': u'36001405fb81b00707464d729f24ff6c3', 'type': 'block'} connect_volume /usr/lib/python2.7/site-packages/nova/virt/libvirt/volume/iscsi.py:65
2019-03-29 10:07:36.080 10556 DEBUG nova.virt.libvirt.driver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Plugging VIFs before live migration. pre_live_migration /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:6797
2019-03-29 10:07:36.081 10556 DEBUG nova.virt.libvirt.vif [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] vif_type=ovs instance=Instance(access_ip_v4=None,access_ip_v6=None,architecture='x86_64',auto_disk_config=True,availability_zone='nova',cell_name=None,cleaned=False,config_drive='True',created_at=2019-03-29T02:02:33Z,default_ephemeral_device=None,default_swap_device=None,deleted=False,deleted_at=None,device_metadata=<?>,disable_terminate=False,display_description='vm001',display_name='vm001',ec2_ids=<?>,ephemeral_gb=0,ephemeral_key_uuid=None,fault=<?>,flavor=Flavor(1),host='com58',hostname='vm001',id=24,image_ref='',info_cache=InstanceInfoCache,instance_type_id=1,kernel_id='',key_data=None,key_name=None,keypairs=<?>,launch_index=0,launched_at=2019-03-29T02:02:45Z,launched_on='com58',locked=False,locked_by=None,memory_mb=512,metadata={},migration_context=<?>,new_flavor=None,node='com58',numa_topology=<?>,old_flavor=None,os_type=None,pci_devices=<?>,pci_requests=<?>,power_state=1,progress=0,project_id='d4bbd972ca1b463ba028ccd1df983917',ramdisk_id='',reservation_id='r-x56iqcqr',root_device_name='/dev/vda',root_gb=1,security_groups=SecurityGroupList,services=<?>,shutdown_terminate=False,system_metadata={boot_roles='admin',image_architecture='x86_64',image_base_image_ref='',image_min_disk='1',image_min_ram='0'},tags=<?>,task_state='migrating',terminated_at=None,updated_at=2019-03-29T02:06:59Z,user_data=None,user_id='9e9e53917dda421083fa6af425c6bfbf',uuid=f5a19c94-1a45-4ea0-9386-77c10705c825,vcpu_model=<?>,vcpus=1,vm_mode=None,vm_state='active') vif={"profile": {}, "ovs_interfaceid": "b3249515-3aab-439f-ba9c-bc498756871a", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"version": 4, "vif_mac": "fa:16:3e:29:72:d6", "floating_ips": [], "label": "net10", "meta": {}, "address": "10.10.10.11", "type": "fixed"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tapb3249515-3a", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:29:72:d6", "active": true, "type": "ovs", "id": "b3249515-3aab-439f-ba9c-bc498756871a", "qbg_params": null} plug /usr/lib/python2.7/site-packages/nova/virt/libvirt/vif.py:776
2019-03-29 10:07:36.082 10556 DEBUG nova.network.os_vif_util [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Converting VIF {"profile": {}, "ovs_interfaceid": "b3249515-3aab-439f-ba9c-bc498756871a", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"version": 4, "vif_mac": "fa:16:3e:29:72:d6", "floating_ips": [], "label": "net10", "meta": {}, "address": "10.10.10.11", "type": "fixed"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tapb3249515-3a", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:29:72:d6", "active": true, "type": "ovs", "id": "b3249515-3aab-439f-ba9c-bc498756871a", "qbg_params": null} nova_to_osvif_vif /usr/lib/python2.7/site-packages/nova/network/os_vif_util.py:413
2019-03-29 10:07:36.083 10556 DEBUG nova.network.os_vif_util [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Converted object VIFBridge(active=True,address=fa:16:3e:29:72:d6,bridge_name='qbrb3249515-3a',has_traffic_filtering=True,id=b3249515-3aab-439f-ba9c-bc498756871a,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tapb3249515-3a') nova_to_osvif_vif /usr/lib/python2.7/site-packages/nova/network/os_vif_util.py:425
2019-03-29 10:07:36.084 10556 DEBUG os_vif [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Plugging vif VIFBridge(active=True,address=fa:16:3e:29:72:d6,bridge_name='qbrb3249515-3a',has_traffic_filtering=True,id=b3249515-3aab-439f-ba9c-bc498756871a,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tapb3249515-3a') plug /usr/lib/python2.7/site-packages/os_vif/__init__.py:79
2019-03-29 10:07:36.085 10556 INFO oslo.privsep.daemon [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Running privsep helper: ['sudo', 'nova-rootwrap', '/etc/nova/rootwrap.conf', 'privsep-helper', '--config-file', '/usr/share/nova/nova-dist.conf', '--config-file', '/etc/nova/nova.conf', '--privsep_context', 'vif_plug_ovs.privsep.vif_plug', '--privsep_sock_path', '/tmp/tmpgVeRZP/privsep.sock']
2019-03-29 10:07:36.759 10556 INFO oslo.privsep.daemon [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Spawned new privsep daemon via rootwrap
2019-03-29 10:07:36.760 10556 DEBUG oslo.privsep.daemon [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Accepted privsep connection to /tmp/tmpgVeRZP/privsep.sock __init__ /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:313
2019-03-29 10:07:36.761 10556 INFO oslo.privsep.daemon [-] privsep daemon starting
2019-03-29 10:07:36.762 10556 INFO oslo.privsep.daemon [-] privsep process running with uid/gid: 0/0
2019-03-29 10:07:36.762 10556 INFO oslo.privsep.daemon [-] privsep process running with capabilities (eff/prm/inh): CAP_NET_ADMIN/CAP_NET_ADMIN/none
2019-03-29 10:07:36.762 10556 INFO oslo.privsep.daemon [-] privsep daemon running as pid 10703
2019-03-29 10:07:36.763 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (1,) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.763 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (2,) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.765 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'vif_plug_ovs.linux_net.ensure_bridge', (u'qbrb3249515-3a',), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.832 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): brctl addbr qbrb3249515-3a out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.844 10556 DEBUG oslo.privsep.daemon [-] CMD "brctl addbr qbrb3249515-3a" returned: 0 in 0.010s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.844 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): brctl setfd qbrb3249515-3a 0 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.849 10556 DEBUG oslo.privsep.daemon [-] CMD "brctl setfd qbrb3249515-3a 0" returned: 0 in 0.005s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.849 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): brctl stp qbrb3249515-3a off out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.852 10556 DEBUG oslo.privsep.daemon [-] CMD "brctl stp qbrb3249515-3a off" returned: 0 in 0.003s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.852 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): tee /sys/class/net/qbrb3249515-3a/bridge/multicast_snooping out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.855 10556 DEBUG oslo.privsep.daemon [-] CMD "tee /sys/class/net/qbrb3249515-3a/bridge/multicast_snooping" returned: 0 in 0.003s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.855 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): tee /proc/sys/net/ipv6/conf/qbrb3249515-3a/disable_ipv6 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.858 10556 DEBUG oslo.privsep.daemon [-] CMD "tee /proc/sys/net/ipv6/conf/qbrb3249515-3a/disable_ipv6" returned: 0 in 0.003s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.859 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.860 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'vif_plug_ovs.linux_net.create_veth_pair', ('qvbb3249515-3a', 'qvob3249515-3a', 1500), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.860 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link add qvbb3249515-3a type veth peer name qvob3249515-3a out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.872 10556 DEBUG oslo.privsep.daemon [-] CMD "ip link add qvbb3249515-3a type veth peer name qvob3249515-3a" returned: 0 in 0.012s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.872 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qvbb3249515-3a up out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.880 10556 DEBUG oslo.privsep.daemon [-] CMD "ip link set qvbb3249515-3a up" returned: 0 in 0.008s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.880 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qvbb3249515-3a promisc on out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.884 10556 DEBUG oslo.privsep.daemon [-] CMD "ip link set qvbb3249515-3a promisc on" returned: 0 in 0.004s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.884 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qvbb3249515-3a mtu 1500 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.887 10556 DEBUG oslo.privsep.daemon [-] CMD "ip link set qvbb3249515-3a mtu 1500" returned: 0 in 0.004s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.888 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qvob3249515-3a up out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.890 10556 DEBUG oslo.privsep.daemon [-] CMD "ip link set qvob3249515-3a up" returned: 0 in 0.003s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.891 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qvob3249515-3a promisc on out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.895 10556 DEBUG oslo.privsep.daemon [-] CMD "ip link set qvob3249515-3a promisc on" returned: 0 in 0.004s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.895 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qvob3249515-3a mtu 1500 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.898 10556 DEBUG oslo.privsep.daemon [-] CMD "ip link set qvob3249515-3a mtu 1500" returned: 0 in 0.003s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.899 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.899 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'vif_plug_ovs.linux_net.add_bridge_port', (u'qbrb3249515-3a', 'qvbb3249515-3a'), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.900 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qbrb3249515-3a up out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.903 10556 DEBUG oslo.privsep.daemon [-] CMD "ip link set qbrb3249515-3a up" returned: 0 in 0.003s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.903 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): brctl addif qbrb3249515-3a qvbb3249515-3a out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.907 10556 DEBUG oslo.privsep.daemon [-] CMD "brctl addif qbrb3249515-3a qvbb3249515-3a" returned: 0 in 0.003s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.908 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.909 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'vif_plug_ovs.linux_net.ensure_ovs_bridge', (u'br-int', 'system'), {}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.910 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ovs-vsctl -- --may-exist add-br br-int -- set Bridge br-int datapath_type=system out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.923 10556 DEBUG oslo.privsep.daemon [-] CMD "ovs-vsctl -- --may-exist add-br br-int -- set Bridge br-int datapath_type=system" returned: 0 in 0.014s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.923 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.925 10556 DEBUG oslo.privsep.daemon [-] privsep: request[77878608]: (3, 'vif_plug_ovs.linux_net.create_ovs_vif_port', (u'br-int', 'qvob3249515-3a', 'b3249515-3aab-439f-ba9c-bc498756871a', u'fa:16:3e:29:72:d6', 'f5a19c94-1a45-4ea0-9386-77c10705c825', 1500), {'timeout': 120}) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.925 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ovs-vsctl --timeout=120 -- --if-exists del-port qvob3249515-3a -- add-port br-int qvob3249515-3a -- set Interface qvob3249515-3a external-ids:iface-id=b3249515-3aab-439f-ba9c-bc498756871a external-ids:iface-status=active external-ids:attached-mac=fa:16:3e:29:72:d6 external-ids:vm-uuid=f5a19c94-1a45-4ea0-9386-77c10705c825 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.939 10556 DEBUG oslo.privsep.daemon [-] CMD "ovs-vsctl --timeout=120 -- --if-exists del-port qvob3249515-3a -- add-port br-int qvob3249515-3a -- set Interface qvob3249515-3a external-ids:iface-id=b3249515-3aab-439f-ba9c-bc498756871a external-ids:iface-status=active external-ids:attached-mac=fa:16:3e:29:72:d6 external-ids:vm-uuid=f5a19c94-1a45-4ea0-9386-77c10705c825" returned: 0 in 0.014s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.939 10556 DEBUG oslo.privsep.daemon [-] Running cmd (subprocess): ip link set qvob3249515-3a mtu 1500 out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.947 10556 DEBUG oslo.privsep.daemon [-] CMD "ip link set qvob3249515-3a mtu 1500" returned: 0 in 0.008s out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.948 10556 DEBUG oslo.privsep.daemon [-] privsep: reply[77878608]: (4, None) out_of_band /usr/lib/python2.7/site-packages/oslo_privsep/daemon.py:194
2019-03-29 10:07:36.948 10556 INFO os_vif [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Successfully plugged vif VIFBridge(active=True,address=fa:16:3e:29:72:d6,bridge_name='qbrb3249515-3a',has_traffic_filtering=True,id=b3249515-3aab-439f-ba9c-bc498756871a,network=Network(968d1118-8921-4ef9-9cf6-534ff3d8bf7d),plugin='ovs',port_profile=VIFPortProfileBase,preserve_on_delete=False,vif_name='tapb3249515-3a')
2019-03-29 10:07:36.949 10556 DEBUG nova.compute.manager [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] driver pre_live_migration data is LibvirtLiveMigrateData(bdms=[LibvirtLiveMigrateBDMInfo],block_migration=True,disk_available_mb=45056,disk_over_commit=False,filename='tmpdjYwQ4',graphics_listen_addr_spice=127.0.0.1,graphics_listen_addr_vnc=0.0.0.0,image_type='default',instance_relative_path='f5a19c94-1a45-4ea0-9386-77c10705c825',is_shared_block_storage=False,is_shared_instance_path=False,is_volume_backed=True,migration=<?>,serial_listen_addr=None,serial_listen_ports=[],supported_perf_events=[],target_connect_addr=None) pre_live_migration /usr/lib/python2.7/site-packages/nova/compute/manager.py:5355
2019-03-29 10:07:36.960 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/extensions.json used request id req-95eab4f9-9617-4b69-a3f9-351ce3f095e6 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:07:37.025 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&binding%3Ahost_id=com58&device_id=f5a19c94-1a45-4ea0-9386-77c10705c825 used request id req-e053e80f-ab11-4c70-9cdf-bda1bad9c93e _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:07:37.734 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] PUT call to neutron for http://172.30.126.56:9696/v2.0/ports/b3249515-3aab-439f-ba9c-bc498756871a.json used request id req-c7944d72-6ace-4fe9-a685-809f475d3496 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:07:37.734 10556 DEBUG nova.network.neutronv2.api [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Port b3249515-3aab-439f-ba9c-bc498756871a updated with migration profile {'migrating_to': 'com57'} successfully _setup_migration_port_profile /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:301
2019-03-29 10:07:37.735 10556 WARNING nova.image.glance [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 10:07:37.736 10556 DEBUG nova.compute.manager [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] pre_live_migration result data is LibvirtLiveMigrateData(bdms=[LibvirtLiveMigrateBDMInfo],block_migration=True,disk_available_mb=45056,disk_over_commit=False,filename='tmpdjYwQ4',graphics_listen_addr_spice=127.0.0.1,graphics_listen_addr_vnc=0.0.0.0,image_type='default',instance_relative_path='f5a19c94-1a45-4ea0-9386-77c10705c825',is_shared_block_storage=False,is_shared_instance_path=False,is_volume_backed=True,migration=<?>,serial_listen_addr=None,serial_listen_ports=[],supported_perf_events=[],target_connect_addr=None) pre_live_migration /usr/lib/python2.7/site-packages/nova/compute/manager.py:5379
2019-03-29 10:07:37.738 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 0ce6c4fc368e4aa9b9275aa8176e9eeb exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:37.862 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 0ce6c4fc368e4aa9b9275aa8176e9eeb __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:37.863 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] sending reply msg_id: 15077abfce144467997b439fb7a67be2 reply queue: reply_b16e68cc3da145ef95ee7086e953eac7 time elapsed: 7.99270364904s _send_reply /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:130
2019-03-29 10:07:38.257 10556 DEBUG nova.virt.driver [-] Emitting event <LifecycleEvent: 1553825258.26, f5a19c94-1a45-4ea0-9386-77c10705c825 => Started> emit_event /usr/lib/python2.7/site-packages/nova/virt/driver.py:1438
2019-03-29 10:07:38.258 10556 INFO nova.compute.manager [-] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] VM Started (Lifecycle Event)
2019-03-29 10:07:38.259 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] CALL msg_id: d8fbbd8a9beb40f29c26fce91bd49e76 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:38.306 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: d8fbbd8a9beb40f29c26fce91bd49e76 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:38.307 10556 DEBUG nova.compute.manager [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176
2019-03-29 10:07:44.317 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 80a01860857a410eaffa0ca20237f321 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:07:46.328 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 80a01860857a410eaffa0ca20237f321 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:07:54.320 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 0c08eb85cd934b4baebca6f8d2bd17e7 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:01.404 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 0c08eb85cd934b4baebca6f8d2bd17e7 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:04.317 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: e0c71fa04576462d8cfe638d1e391471 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:09.204 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: e0c71fa04576462d8cfe638d1e391471 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:11.595 10556 DEBUG nova.virt.driver [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] Emitting event <LifecycleEvent: 1553825291.59, f5a19c94-1a45-4ea0-9386-77c10705c825 => Resumed> emit_event /usr/lib/python2.7/site-packages/nova/virt/driver.py:1438
2019-03-29 10:08:11.596 10556 INFO nova.compute.manager [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] VM Resumed (Lifecycle Event)
2019-03-29 10:08:11.597 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] CALL msg_id: b787257893fb470e966eb28e004e8d76 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:12.785 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: b787257893fb470e966eb28e004e8d76 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:12.786 10556 DEBUG nova.compute.manager [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176
2019-03-29 10:08:12.789 10556 DEBUG nova.compute.manager [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Synchronizing instance power state after lifecycle event "Resumed"; current vm_state: active, current task_state: migrating, current DB power_state: 1, VM power_state: 1 handle_lifecycle_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:1088
2019-03-29 10:08:12.792 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] CALL msg_id: a08776657151493dada1a84798334923 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:12.836 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: a08776657151493dada1a84798334923 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:12.837 10556 INFO nova.compute.manager [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] During the sync_power process the instance has moved from host com58 to host com57
2019-03-29 10:08:12.838 10556 DEBUG nova.virt.driver [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] Emitting event <LifecycleEvent: 1553825291.6, f5a19c94-1a45-4ea0-9386-77c10705c825 => Resumed> emit_event /usr/lib/python2.7/site-packages/nova/virt/driver.py:1438
2019-03-29 10:08:12.839 10556 INFO nova.compute.manager [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] VM Resumed (Lifecycle Event)
2019-03-29 10:08:12.840 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] CALL msg_id: e4d2b15e4fe94008aba49f5cfb4ad85d exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:12.883 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: e4d2b15e4fe94008aba49f5cfb4ad85d __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:12.885 10556 DEBUG nova.compute.manager [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176
2019-03-29 10:08:12.888 10556 DEBUG nova.compute.manager [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Synchronizing instance power state after lifecycle event "Resumed"; current vm_state: active, current task_state: migrating, current DB power_state: 1, VM power_state: 1 handle_lifecycle_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:1088
2019-03-29 10:08:12.890 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] CALL msg_id: df0ca50e6ee74e5181043702e36818c6 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:12.936 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: df0ca50e6ee74e5181043702e36818c6 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:12.937 10556 INFO nova.compute.manager [req-c7b3a937-5bf4-44c3-9927-d7d0198b4e6f - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] During the sync_power process the instance has moved from host com58 to host com57
2019-03-29 10:08:14.317 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: c56b047996a4459b8331eb3bd9dc9cbe exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:19.434 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_rebooting_instances run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:08:19.435 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._reclaim_queued_deletes run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:08:19.436 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CONF.reclaim_instance_interval <= 0, skipping... _reclaim_queued_deletes /usr/lib/python2.7/site-packages/nova/compute/manager.py:6556
2019-03-29 10:08:21.437 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._instance_usage_audit run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:08:23.432 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._check_instance_build_time run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:08:23.536 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: c56b047996a4459b8331eb3bd9dc9cbe __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:24.318 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 2c77d2182dae4642b26fd11b9b2e943c exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:24.433 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._sync_scheduler_instance_info run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:08:24.434 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 0c9b2647c2a64ef0bd3e8310a3f20e6a exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:24.560 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 0c9b2647c2a64ef0bd3e8310a3f20e6a __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:24.562 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CAST unique_id: b5f49b04669441058ce02261fc40403b FANOUT topic 'scheduler' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:539
2019-03-29 10:08:24.563 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._heal_instance_info_cache run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:08:24.564 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Starting heal instance info cache _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5884
2019-03-29 10:08:24.564 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Rebuilding the list of instances to heal _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5888
2019-03-29 10:08:24.565 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: de63ad1574094fd7bbd9ab812e25e189 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:24.691 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: de63ad1574094fd7bbd9ab812e25e189 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:24.692 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Didn't find any instances for network info cache update. _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5960
2019-03-29 10:08:25.563 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager.update_available_resource run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:08:25.565 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 59c6991c42a449738cc98326bf962c90 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:25.688 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 59c6991c42a449738cc98326bf962c90 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:25.691 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Auditing locally available compute resources for com57 (node: com57) update_available_resource /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:534
2019-03-29 10:08:25.704 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: e12320d536934e1bb178a757afa34a49 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:25.931 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: e12320d536934e1bb178a757afa34a49 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:25.933 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 548a5174abf246f8937db12bedb7290d exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:26.044 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 548a5174abf246f8937db12bedb7290d __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:26.047 10556 DEBUG nova.virt.libvirt.driver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] skipping disk /dev/disk/by-path/ip-172.30.126.56:3260-iscsi-iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e-lun-0 (vda) as it is a volume _get_instance_disk_info /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:7082
2019-03-29 10:08:26.134 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Hypervisor: free VCPUs: 1 _report_hypervisor_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:655
2019-03-29 10:08:26.135 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Hypervisor/Node resource view: name=com57 free_ram=6990MB free_disk=44GB free_vcpus=1 pci_devices=[{"dev_id": "pci_0000_00_00_0", "product_id": "7190", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7190", "address": "0000:00:00.0"}, {"dev_id": "pci_0000_00_01_0", "product_id": "7191", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7191", "address": "0000:00:01.0"}, {"dev_id": "pci_0000_00_07_0", "product_id": "7110", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7110", "address": "0000:00:07.0"}, {"dev_id": "pci_0000_00_07_1", "product_id": "7111", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7111", "address": "0000:00:07.1"}, {"dev_id": "pci_0000_00_07_3", "product_id": "7113", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7113", "address": "0000:00:07.3"}, {"dev_id": "pci_0000_00_07_7", "product_id": "0740", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0740", "address": "0000:00:07.7"}, {"dev_id": "pci_0000_00_0f_0", "product_id": "0405", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0405", "address": "0000:00:0f.0"}, {"dev_id": "pci_0000_00_11_0", "product_id": "0790", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0790", "address": "0000:00:11.0"}, {"dev_id": "pci_0000_00_15_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.0"}, {"dev_id": "pci_0000_03_00_0", "product_id": "07c0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07c0", "address": "0000:03:00.0"}, {"dev_id": "pci_0000_00_15_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.1"}, {"dev_id": "pci_0000_00_15_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.2"}, {"dev_id": "pci_0000_00_15_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.3"}, {"dev_id": "pci_0000_00_15_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.4"}, {"dev_id": "pci_0000_00_15_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.5"}, {"dev_id": "pci_0000_00_15_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.6"}, {"dev_id": "pci_0000_00_15_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.7"}, {"dev_id": "pci_0000_00_16_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.0"}, {"dev_id": "pci_0000_0b_00_0", "product_id": "07b0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07b0", "address": "0000:0b:00.0"}, {"dev_id": "pci_0000_00_16_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.1"}, {"dev_id": "pci_0000_00_16_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.2"}, {"dev_id": "pci_0000_00_16_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.3"}, {"dev_id": "pci_0000_00_16_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.4"}, {"dev_id": "pci_0000_00_16_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.5"}, {"dev_id": "pci_0000_00_16_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.6"}, {"dev_id": "pci_0000_00_16_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.7"}, {"dev_id": "pci_0000_00_17_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.0"}, {"dev_id": "pci_0000_13_00_0", "product_id": "07b0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07b0", "address": "0000:13:00.0"}, {"dev_id": "pci_0000_00_17_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.1"}, {"dev_id": "pci_0000_00_17_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.2"}, {"dev_id": "pci_0000_00_17_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.3"}, {"dev_id": "pci_0000_00_17_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.4"}, {"dev_id": "pci_0000_00_17_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.5"}, {"dev_id": "pci_0000_00_17_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.6"}, {"dev_id": "pci_0000_00_17_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.7"}, {"dev_id": "pci_0000_00_18_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.0"}, {"dev_id": "pci_0000_00_18_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.1"}, {"dev_id": "pci_0000_00_18_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.2"}, {"dev_id": "pci_0000_00_18_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.3"}, {"dev_id": "pci_0000_00_18_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.4"}, {"dev_id": "pci_0000_00_18_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.5"}, {"dev_id": "pci_0000_00_18_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.6"}, {"dev_id": "pci_0000_00_18_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.7"}] _report_hypervisor_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:672
2019-03-29 10:08:26.135 10556 DEBUG oslo_concurrency.lockutils [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Lock "compute_resources" acquired by "nova.compute.resource_tracker._update_available_resource" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 10:08:26.137 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 540d49a3901a4522b7851f2eecb3a635 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:43.739 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 2c77d2182dae4642b26fd11b9b2e943c __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:43.740 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 540d49a3901a4522b7851f2eecb3a635 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:43.741 10556 WARNING oslo.service.loopingcall [-] Function 'nova.servicegroup.drivers.db.DbDriver._report_state' run outlasted interval by 9.42 sec
2019-03-29 10:08:43.741 10556 DEBUG nova.scheduler.client.report [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Refreshing aggregate associations for resource provider db732609-7867-486c-8e55-f52f70acfa62 _ensure_resource_provider /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:402
2019-03-29 10:08:43.744 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 0b900d3b27d7490982437bcf28e18098 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:43.785 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 564af3a04c3c4f4b8e9583b495ead854 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:43.804 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 564af3a04c3c4f4b8e9583b495ead854 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:43.820 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 05a5d9ec0be74abeb28c11d060a4f835 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:43.871 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 05a5d9ec0be74abeb28c11d060a4f835 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:43.872 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Total usable vcpus: 2, total allocated vcpus: 0 _report_final_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:688
2019-03-29 10:08:43.873 10556 INFO nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Final resource view: name=com57 phys_ram=8191MB used_ram=512MB phys_disk=47GB used_disk=0GB total_vcpus=2 used_vcpus=0 pci_stats=[]
2019-03-29 10:08:43.876 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Compute_service record updated for com57:com57 _update_available_resource /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:626
2019-03-29 10:08:43.876 10556 DEBUG oslo_concurrency.lockutils [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Lock "compute_resources" released by "nova.compute.resource_tracker._update_available_resource" :: held 17.741s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 10:08:43.877 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_unconfirmed_resizes run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:08:43.877 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_rescued_instances run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:08:44.748 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_volume_usage run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:08:46.041 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 0b900d3b27d7490982437bcf28e18098 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:50.152 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received message msg_id: 86b1b3ead30d4da6ae6be20c5b641973 reply to reply_b16e68cc3da145ef95ee7086e953eac7 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:255
2019-03-29 10:08:50.155 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 93718647c7394232883f1d1c9f3abd06 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:50.186 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 93718647c7394232883f1d1c9f3abd06 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:50.187 10556 INFO nova.compute.manager [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Post operation of migration started
2019-03-29 10:08:50.251 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&binding%3Ahost_id=com58&device_id=f5a19c94-1a45-4ea0-9386-77c10705c825 used request id req-fd729ef0-5cbe-4e65-b3de-a0de5c60f2aa _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:50.562 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] PUT call to neutron for http://172.30.126.56:9696/v2.0/ports/b3249515-3aab-439f-ba9c-bc498756871a.json used request id req-b7cc3e50-0d43-42cd-aa39-f86d4967a9af _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:50.562 10556 DEBUG nova.network.neutronv2.api [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Port b3249515-3aab-439f-ba9c-bc498756871a updated with migration profile {u'migrating_to': 'com57'} successfully _setup_migration_port_profile /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:301
2019-03-29 10:08:50.750 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=f5a19c94-1a45-4ea0-9386-77c10705c825 used request id req-a46b5053-9eb5-4f04-896e-6619720bba4c _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:50.750 10556 INFO nova.network.neutronv2.api [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Updating port b3249515-3aab-439f-ba9c-bc498756871a with attributes {'binding:profile': {}, 'binding:host_id': 'com57'}
2019-03-29 10:08:51.926 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] PUT call to neutron for http://172.30.126.56:9696/v2.0/ports/b3249515-3aab-439f-ba9c-bc498756871a.json used request id req-96fc6f53-9d4c-45cd-b80d-310d318eeb63 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:51.926 10556 DEBUG oslo_concurrency.lockutils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Acquired semaphore "refresh_cache-f5a19c94-1a45-4ea0-9386-77c10705c825" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2019-03-29 10:08:51.927 10556 DEBUG nova.network.neutronv2.api [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] _get_instance_nw_info() _get_instance_nw_info /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1295
2019-03-29 10:08:51.928 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 85b52c3dbced4d5a87f2a038a95dc2c4 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:51.938 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 85b52c3dbced4d5a87f2a038a95dc2c4 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:51.993 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=f5a19c94-1a45-4ea0-9386-77c10705c825 used request id req-efa632b7-79d5-4ae9-862b-a41a14e5b95f _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:52.212 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/networks.json?id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d used request id req-becc6d40-7211-419a-8685-cbc3083052d7 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:52.232 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/floatingips.json?fixed_ip_address=10.10.10.11&port_id=b3249515-3aab-439f-ba9c-bc498756871a used request id req-bf7eee60-3175-49ed-bb49-198fc5a645c1 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:52.290 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/subnets.json?id=b9f5b815-a0ab-4476-a216-6b824fda2957 used request id req-8fc851d7-4372-4cb5-b64d-4a8ac230680e _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:52.331 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?network_id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d&device_owner=network%3Adhcp used request id req-35ea3667-59f7-4b91-b44f-2528d244c043 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:52.331 10556 DEBUG nova.network.base_api [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Updating instance_info_cache with network_info: [{"profile": {}, "ovs_interfaceid": "b3249515-3aab-439f-ba9c-bc498756871a", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.11"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tapb3249515-3a", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:29:72:d6", "active": false, "type": "ovs", "id": "b3249515-3aab-439f-ba9c-bc498756871a", "qbg_params": null}] update_instance_cache_with_nw_info /usr/lib/python2.7/site-packages/nova/network/base_api.py:48
2019-03-29 10:08:52.333 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 78eb96ed8aa44b7da199aca1ea16eb31 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:52.358 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 78eb96ed8aa44b7da199aca1ea16eb31 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:52.360 10556 DEBUG oslo_concurrency.lockutils [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] Releasing semaphore "refresh_cache-f5a19c94-1a45-4ea0-9386-77c10705c825" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2019-03-29 10:08:52.360 10556 WARNING nova.image.glance [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 10:08:52.364 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: 329a350552c7450194bf7dc248a28531 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:52.374 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 329a350552c7450194bf7dc248a28531 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:52.431 10556 DEBUG nova.compute.manager [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Checking state _get_power_state /usr/lib/python2.7/site-packages/nova/compute/manager.py:1176
2019-03-29 10:08:52.435 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: bfce601e752144d080e146c096ba3f6a exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:52.454 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: bfce601e752144d080e146c096ba3f6a __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:52.460 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: aa9caeae1ede4d6081222867a904d20a exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:52.540 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: aa9caeae1ede4d6081222867a904d20a __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:52.611 10556 DEBUG neutronclient.v2_0.client [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&binding%3Ahost_id=com57&device_id=f5a19c94-1a45-4ea0-9386-77c10705c825 used request id req-3cd83179-4763-4560-896a-461dc4c6833d _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:52.612 10556 WARNING nova.image.glance [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] No protocol specified in for api_server 'http://172.30.126.56:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2019-03-29 10:08:52.613 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] CALL msg_id: dcf458fbd14d45a89d5410383bc241e8 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:52.643 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: dcf458fbd14d45a89d5410383bc241e8 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:52.644 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-4c11c178-33fc-4f00-a66d-8796ce896df9 9e9e53917dda421083fa6af425c6bfbf d4bbd972ca1b463ba028ccd1df983917 - - -] sending reply msg_id: 86b1b3ead30d4da6ae6be20c5b641973 reply queue: reply_b16e68cc3da145ef95ee7086e953eac7 time elapsed: 2.491486796s _send_reply /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:130
2019-03-29 10:08:52.787 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: 1a19d91c3f9449818eec196a5f905fcc __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 10:08:52.790 10556 DEBUG nova.compute.manager [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Received event network-changed external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 10:08:52.790 10556 DEBUG oslo_concurrency.lockutils [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Acquired semaphore "refresh_cache-f5a19c94-1a45-4ea0-9386-77c10705c825" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2019-03-29 10:08:52.791 10556 DEBUG nova.network.neutronv2.api [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] _get_instance_nw_info() _get_instance_nw_info /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1295
2019-03-29 10:08:52.792 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] CALL msg_id: 52cfc9ced082449e90e07368ce25977e exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:52.808 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 52cfc9ced082449e90e07368ce25977e __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:52.886 10556 DEBUG neutronclient.v2_0.client [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=f5a19c94-1a45-4ea0-9386-77c10705c825 used request id req-5058238d-6705-4e20-9fa1-fa063e93fda1 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:53.043 10556 DEBUG neutronclient.v2_0.client [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/networks.json?id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d used request id req-cbf3e67e-a510-478d-b638-ccfcd2cb2771 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:53.107 10556 DEBUG neutronclient.v2_0.client [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/floatingips.json?fixed_ip_address=10.10.10.11&port_id=b3249515-3aab-439f-ba9c-bc498756871a used request id req-0d3fc8b9-e5f6-4fe3-a94d-715ec970d74c _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:53.247 10556 DEBUG neutronclient.v2_0.client [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/subnets.json?id=b9f5b815-a0ab-4476-a216-6b824fda2957 used request id req-af17e211-762e-487a-94b3-ca741e0f3eef _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:53.312 10556 DEBUG neutronclient.v2_0.client [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?network_id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d&device_owner=network%3Adhcp used request id req-bf27e669-2906-40b1-bb97-0baf4049c03c _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:08:53.313 10556 DEBUG nova.network.base_api [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Updating instance_info_cache with network_info: [{"profile": {}, "ovs_interfaceid": "b3249515-3aab-439f-ba9c-bc498756871a", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.11"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tapb3249515-3a", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:29:72:d6", "active": false, "type": "ovs", "id": "b3249515-3aab-439f-ba9c-bc498756871a", "qbg_params": null}] update_instance_cache_with_nw_info /usr/lib/python2.7/site-packages/nova/network/base_api.py:48
2019-03-29 10:08:53.315 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] CALL msg_id: 2bbef6d5923f4850b87c864b3eda952b exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:53.327 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 2bbef6d5923f4850b87c864b3eda952b __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:53.328 10556 DEBUG oslo_concurrency.lockutils [req-731a058a-c8b7-43e4-a8be-3f531304cc15 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Releasing semaphore "refresh_cache-f5a19c94-1a45-4ea0-9386-77c10705c825" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2019-03-29 10:08:53.744 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 9adfac15bf0849ae89f0aac133cda13e exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:08:53.807 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 9adfac15bf0849ae89f0aac133cda13e __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:08:54.252 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received message with unique_id: f996f9cf89194bba9d6490f89213d7c1 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:257
2019-03-29 10:08:54.255 10556 DEBUG nova.compute.manager [req-f00850e2-775a-452b-808c-e2776e4e5e88 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Received event network-vif-plugged-b3249515-3aab-439f-ba9c-bc498756871a external_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:6861
2019-03-29 10:08:54.256 10556 DEBUG oslo_concurrency.lockutils [req-f00850e2-775a-452b-808c-e2776e4e5e88 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "f5a19c94-1a45-4ea0-9386-77c10705c825-events" acquired by "nova.compute.manager._pop_event" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 10:08:54.256 10556 DEBUG nova.compute.manager [req-f00850e2-775a-452b-808c-e2776e4e5e88 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Unexpected attempt to pop events during shutdown _pop_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:350
2019-03-29 10:08:54.257 10556 DEBUG oslo_concurrency.lockutils [req-f00850e2-775a-452b-808c-e2776e4e5e88 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] Lock "f5a19c94-1a45-4ea0-9386-77c10705c825-events" released by "nova.compute.manager._pop_event" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 10:08:54.257 10556 DEBUG nova.compute.manager [req-f00850e2-775a-452b-808c-e2776e4e5e88 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] No waiting events found dispatching network-vif-plugged-b3249515-3aab-439f-ba9c-bc498756871a pop_instance_event /usr/lib/python2.7/site-packages/nova/compute/manager.py:366
2019-03-29 10:08:54.258 10556 WARNING nova.compute.manager [req-f00850e2-775a-452b-808c-e2776e4e5e88 92aab8401906424bbc6df9e557586649 a77dc037ca704f3883e8ed733fff997f - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Received unexpected event network-vif-plugged-b3249515-3aab-439f-ba9c-bc498756871a for instance
2019-03-29 10:09:03.739 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: c51d71dc536f4bf8bbc206dbbee50ee2 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:03.770 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: c51d71dc536f4bf8bbc206dbbee50ee2 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:13.743 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 8782dc7b65f6474cbb253d4de9ebe7c8 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:13.777 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 8782dc7b65f6474cbb253d4de9ebe7c8 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:20.434 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._reclaim_queued_deletes run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:09:20.434 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CONF.reclaim_instance_interval <= 0, skipping... _reclaim_queued_deletes /usr/lib/python2.7/site-packages/nova/compute/manager.py:6556
2019-03-29 10:09:21.435 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_rebooting_instances run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:09:23.432 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._check_instance_build_time run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:09:23.434 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._instance_usage_audit run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:09:23.739 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 6e8bfe0957a0472f90bae2c907f103f8 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:23.784 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 6e8bfe0957a0472f90bae2c907f103f8 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:25.434 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._heal_instance_info_cache run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:09:25.434 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Starting heal instance info cache _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5884
2019-03-29 10:09:25.434 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Rebuilding the list of instances to heal _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5888
2019-03-29 10:09:25.436 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: b2e93d9ecc0f42ffb2736fcc38851d4f exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:25.465 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: b2e93d9ecc0f42ffb2736fcc38851d4f __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:25.467 10556 DEBUG oslo_concurrency.lockutils [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Acquired semaphore "refresh_cache-f5a19c94-1a45-4ea0-9386-77c10705c825" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2019-03-29 10:09:25.467 10556 DEBUG nova.network.neutronv2.api [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] _get_instance_nw_info() _get_instance_nw_info /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1295
2019-03-29 10:09:25.467 10556 DEBUG nova.objects.instance [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Lazy-loading 'info_cache' on Instance uuid f5a19c94-1a45-4ea0-9386-77c10705c825 obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1034
2019-03-29 10:09:25.468 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 09f6c07d44fa4a50acd52e7b16ae6f89 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:25.520 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 09f6c07d44fa4a50acd52e7b16ae6f89 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:25.521 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 7efd3438582e44e39e8ba157865aa4bd exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:25.529 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 7efd3438582e44e39e8ba157865aa4bd __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:25.599 10556 DEBUG neutronclient.v2_0.client [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?tenant_id=d4bbd972ca1b463ba028ccd1df983917&device_id=f5a19c94-1a45-4ea0-9386-77c10705c825 used request id req-774f796d-360b-4635-a26e-d750a8f812e9 _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:09:25.716 10556 DEBUG neutronclient.v2_0.client [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/networks.json?id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d used request id req-0a70a212-cb77-4541-89e4-45b276fbf11b _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:09:25.736 10556 DEBUG neutronclient.v2_0.client [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/floatingips.json?fixed_ip_address=10.10.10.11&port_id=b3249515-3aab-439f-ba9c-bc498756871a used request id req-af098b3c-727e-4d96-8017-15e2321347ec _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:09:25.798 10556 DEBUG neutronclient.v2_0.client [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/subnets.json?id=b9f5b815-a0ab-4476-a216-6b824fda2957 used request id req-11988653-150c-4590-9be4-c37c8584a67b _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:09:25.841 10556 DEBUG neutronclient.v2_0.client [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] GET call to neutron for http://172.30.126.56:9696/v2.0/ports.json?network_id=968d1118-8921-4ef9-9cf6-534ff3d8bf7d&device_owner=network%3Adhcp used request id req-ee4f1989-9370-4200-8713-2c3043c2caff _append_request_id /usr/lib/python2.7/site-packages/neutronclient/v2_0/client.py:128
2019-03-29 10:09:25.842 10556 DEBUG nova.network.base_api [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Updating instance_info_cache with network_info: [{"profile": {}, "ovs_interfaceid": "b3249515-3aab-439f-ba9c-bc498756871a", "preserve_on_delete": false, "network": {"bridge": "br-int", "subnets": [{"ips": [{"meta": {}, "version": 4, "type": "fixed", "floating_ips": [], "address": "10.10.10.11"}], "version": 4, "meta": {}, "dns": [], "routes": [], "cidr": "10.10.10.0/24", "gateway": {"meta": {}, "version": 4, "type": "gateway", "address": "10.10.10.1"}}], "meta": {"injected": false, "tenant_id": "d4bbd972ca1b463ba028ccd1df983917", "mtu": 1500}, "id": "968d1118-8921-4ef9-9cf6-534ff3d8bf7d", "label": "net10"}, "devname": "tapb3249515-3a", "vnic_type": "normal", "qbh_params": null, "meta": {}, "details": {"port_filter": true, "ovs_hybrid_plug": true}, "address": "fa:16:3e:29:72:d6", "active": true, "type": "ovs", "id": "b3249515-3aab-439f-ba9c-bc498756871a", "qbg_params": null}] update_instance_cache_with_nw_info /usr/lib/python2.7/site-packages/nova/network/base_api.py:48
2019-03-29 10:09:25.843 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: ed8fc2945bb34c6fb9b34563a4a1b820 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:25.940 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: ed8fc2945bb34c6fb9b34563a4a1b820 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:25.941 10556 DEBUG oslo_concurrency.lockutils [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Releasing semaphore "refresh_cache-f5a19c94-1a45-4ea0-9386-77c10705c825" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2019-03-29 10:09:25.941 10556 DEBUG nova.compute.manager [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Updated the network info_cache for instance _heal_instance_info_cache /usr/lib/python2.7/site-packages/nova/compute/manager.py:5946
2019-03-29 10:09:26.941 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_volume_usage run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:09:26.942 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_rescued_instances run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:09:27.435 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager.update_available_resource run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:09:27.437 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: bdc287cdf0bf47c0b9317c6a7ac41de8 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:27.457 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: bdc287cdf0bf47c0b9317c6a7ac41de8 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:27.461 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Auditing locally available compute resources for com57 (node: com57) update_available_resource /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:534
2019-03-29 10:09:27.473 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: bfc983d800a24e21a0d8bc843ccdea93 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:27.504 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: bfc983d800a24e21a0d8bc843ccdea93 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:27.506 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 9c1995f1ba914a24bff9a60edaf9f6ef exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:27.516 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 9c1995f1ba914a24bff9a60edaf9f6ef __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:27.518 10556 DEBUG nova.virt.libvirt.driver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] skipping disk /dev/disk/by-path/ip-172.30.126.56:3260-iscsi-iqn.2010-10.org.openstack:volume-78a56ddc-ee21-4278-991e-17b30d68fa6e-lun-0 (vda) as it is a volume _get_instance_disk_info /usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py:7082
2019-03-29 10:09:27.603 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Hypervisor: free VCPUs: 1 _report_hypervisor_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:655
2019-03-29 10:09:27.604 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Hypervisor/Node resource view: name=com57 free_ram=6990MB free_disk=44GB free_vcpus=1 pci_devices=[{"dev_id": "pci_0000_00_00_0", "product_id": "7190", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7190", "address": "0000:00:00.0"}, {"dev_id": "pci_0000_00_01_0", "product_id": "7191", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7191", "address": "0000:00:01.0"}, {"dev_id": "pci_0000_00_07_0", "product_id": "7110", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7110", "address": "0000:00:07.0"}, {"dev_id": "pci_0000_00_07_1", "product_id": "7111", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7111", "address": "0000:00:07.1"}, {"dev_id": "pci_0000_00_07_3", "product_id": "7113", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "8086", "label": "label_8086_7113", "address": "0000:00:07.3"}, {"dev_id": "pci_0000_00_07_7", "product_id": "0740", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0740", "address": "0000:00:07.7"}, {"dev_id": "pci_0000_00_0f_0", "product_id": "0405", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0405", "address": "0000:00:0f.0"}, {"dev_id": "pci_0000_00_11_0", "product_id": "0790", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_0790", "address": "0000:00:11.0"}, {"dev_id": "pci_0000_00_15_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.0"}, {"dev_id": "pci_0000_03_00_0", "product_id": "07c0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07c0", "address": "0000:03:00.0"}, {"dev_id": "pci_0000_00_15_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.1"}, {"dev_id": "pci_0000_00_15_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.2"}, {"dev_id": "pci_0000_00_15_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.3"}, {"dev_id": "pci_0000_00_15_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.4"}, {"dev_id": "pci_0000_00_15_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.5"}, {"dev_id": "pci_0000_00_15_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.6"}, {"dev_id": "pci_0000_00_15_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:15.7"}, {"dev_id": "pci_0000_00_16_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.0"}, {"dev_id": "pci_0000_0b_00_0", "product_id": "07b0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07b0", "address": "0000:0b:00.0"}, {"dev_id": "pci_0000_00_16_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.1"}, {"dev_id": "pci_0000_00_16_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.2"}, {"dev_id": "pci_0000_00_16_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.3"}, {"dev_id": "pci_0000_00_16_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.4"}, {"dev_id": "pci_0000_00_16_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.5"}, {"dev_id": "pci_0000_00_16_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.6"}, {"dev_id": "pci_0000_00_16_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:16.7"}, {"dev_id": "pci_0000_00_17_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.0"}, {"dev_id": "pci_0000_13_00_0", "product_id": "07b0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07b0", "address": "0000:13:00.0"}, {"dev_id": "pci_0000_00_17_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.1"}, {"dev_id": "pci_0000_00_17_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.2"}, {"dev_id": "pci_0000_00_17_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.3"}, {"dev_id": "pci_0000_00_17_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.4"}, {"dev_id": "pci_0000_00_17_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.5"}, {"dev_id": "pci_0000_00_17_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.6"}, {"dev_id": "pci_0000_00_17_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:17.7"}, {"dev_id": "pci_0000_00_18_0", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.0"}, {"dev_id": "pci_0000_00_18_1", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.1"}, {"dev_id": "pci_0000_00_18_2", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.2"}, {"dev_id": "pci_0000_00_18_3", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.3"}, {"dev_id": "pci_0000_00_18_4", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.4"}, {"dev_id": "pci_0000_00_18_5", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.5"}, {"dev_id": "pci_0000_00_18_6", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.6"}, {"dev_id": "pci_0000_00_18_7", "product_id": "07a0", "dev_type": "type-PCI", "numa_node": null, "vendor_id": "15ad", "label": "label_15ad_07a0", "address": "0000:00:18.7"}] _report_hypervisor_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:672
2019-03-29 10:09:27.604 10556 DEBUG oslo_concurrency.lockutils [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Lock "compute_resources" acquired by "nova.compute.resource_tracker._update_available_resource" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2019-03-29 10:09:27.606 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 6501db9126e4484d82273d4b5732f8b0 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:27.697 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 6501db9126e4484d82273d4b5732f8b0 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:27.698 10556 DEBUG nova.scheduler.client.report [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Refreshing aggregate associations for resource provider db732609-7867-486c-8e55-f52f70acfa62 _ensure_resource_provider /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:402
2019-03-29 10:09:27.732 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 40d65119d41f4b9aa9bfabe1c7a9b618 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:27.759 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 40d65119d41f4b9aa9bfabe1c7a9b618 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:27.763 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: a31002bdbcde456db2f50c5159030f04 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:27.774 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: a31002bdbcde456db2f50c5159030f04 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:27.789 10556 DEBUG nova.scheduler.client.report [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Sending allocation for instance {'MEMORY_MB': 512, 'VCPU': 1} _allocate_for_instance /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:664
2019-03-29 10:09:27.838 10556 INFO nova.scheduler.client.report [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] [instance: f5a19c94-1a45-4ea0-9386-77c10705c825] Submitted allocation for instance
2019-03-29 10:09:27.871 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 112d33216d404585ab7d877e3186ed63 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:27.905 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 112d33216d404585ab7d877e3186ed63 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:27.906 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Total usable vcpus: 2, total allocated vcpus: 1 _report_final_resource_view /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:688
2019-03-29 10:09:27.906 10556 INFO nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Final resource view: name=com57 phys_ram=8191MB used_ram=1024MB phys_disk=47GB used_disk=1GB total_vcpus=2 used_vcpus=1 pci_stats=[]
2019-03-29 10:09:27.910 10556 DEBUG oslo_messaging._drivers.amqpdriver [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] CALL msg_id: 022f6f239ea44434a13f64a967dc0a05 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:27.936 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 022f6f239ea44434a13f64a967dc0a05 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:27.937 10556 DEBUG nova.scheduler.client.report [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Refreshing aggregate associations for resource provider db732609-7867-486c-8e55-f52f70acfa62 _ensure_resource_provider /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:402
2019-03-29 10:09:27.971 10556 DEBUG nova.scheduler.client.report [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Updating our resource provider generation from 32 to 33 _get_inventory_and_update_provider_generation /usr/lib/python2.7/site-packages/nova/scheduler/client/report.py:445
2019-03-29 10:09:27.972 10556 DEBUG nova.compute.resource_tracker [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Compute_service record updated for com57:com57 _update_available_resource /usr/lib/python2.7/site-packages/nova/compute/resource_tracker.py:626
2019-03-29 10:09:27.972 10556 DEBUG oslo_concurrency.lockutils [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Lock "compute_resources" released by "nova.compute.resource_tracker._update_available_resource" :: held 0.368s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2019-03-29 10:09:29.434 10556 DEBUG oslo_service.periodic_task [req-8886df63-1d00-4f26-87b4-d90c18c712a7 - - - - -] Running periodic task ComputeManager._poll_unconfirmed_resizes run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2019-03-29 10:09:33.737 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: ceffea38aa0a45efa724bc4afb73a442 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:33.777 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: ceffea38aa0a45efa724bc4afb73a442 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:43.742 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: 6c7f248c3f4c416dbb8dcb5dd53a0727 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:43.785 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: 6c7f248c3f4c416dbb8dcb5dd53a0727 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407
2019-03-29 10:09:53.747 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] CALL msg_id: b8191ac126d2411b99e19fe835352f46 exchange 'nova' topic 'conductor' _send /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:550
2019-03-29 10:09:53.782 10556 DEBUG oslo_messaging._drivers.amqpdriver [-] received reply msg_id: b8191ac126d2411b99e19fe835352f46 __call__ /usr/lib/python2.7/site-packages/oslo_messaging/_drivers/amqpdriver.py:407

